cmd/nri-kubernetes/main.go

```diff
@@ -73,19 +73,28 @@ func main() {
 		}
 	}
 
-	iw, err := integration.NewWrapper(
+	integrationOptions := []integration.OptionFunc{
 		integration.WithLogger(logger),
 		integration.WithMetadata(integration.Metadata{
 			Name:    integrationName,
 			Version: integrationVersion,
 		}),
-	)
+	}
+
+	fileInfo, err := os.Stdout.Stat()
+	if err == nil && fileInfo.Mode()&os.ModeCharDevice != 0 {
```

Comment:
a bit too hacky, no? 😅  could we add a parameter like HTTPsink.enable?

Reply:
Fair enough, I think a config flag + env var would be more easier to understand.

Reply:
Done!